### DISCUSSION TITLE
Please insert the product

### DISCUSSION THREAD

**Discussion by aleduni** [tesla-android/issue-tracker#284] [Q&A]
I just installed the application on ny Pi following the instructions, the Tesla recognize the Pi, but the Pi not recognize the dongle, this dongle in a car with android auto works perfect is a Carlinkit 3.0 CPC200-U2W plus.
any suggestion?

thanks.

**Comment by mikegapinski** [comment]
Wrong dongle modem. It needs to end with CCPA
